interview-questions.md

```diff
@@ -9,41 +9,41 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: `this` is used when refering to a particular key:value pair in an object. For example, to refer to a key:value pair in an object, you will need to call on it by using `this` like so; "this.state.colors" , "this.state.newColor". 
 
-  Researched answer:
+  Researched answer: A keyword that refers to the object it belongs to. Can be used in methods, classes, and functions. 
 
 
 
 2. What is yarn? What file(s) are modified in a React application when you run yarn?
 
-  Your answer:
+  Your answer: yarn allows us to store and test our code. I believe the files that are motified when you run yarn are the node.module files that are found in a react application.
 
-  Researched answer:
+  Researched answer: yarn is a package manager for your code. Yarn generates a "yarn.lock" file that helps with merging. Yarn installs and stores dependencies locally and fetches them with a "yarn add" command
 
 
 
 3. What are props in React?
 
-  Your answer:
+  Your answer: props in React allow us to pass down properties from the parent class to a child class. For example, if we have a parent class with a "this.carColor = red" once we pass props in a new child class, the "this.carColor = red" will get passed automatically.
```

Comment:
Yea, if we swap put the word class here with component (slight but important difference) the you have it spot on! The child component will call this.props.carColor === Red? True